### ISSUE TITLE
Deepmind Hyperparameters Results in Low Accuracy

### ISSUE THREAD

**Issue by ezshen** [timchen0618/pytorch-leo#1]
Thank you for this great work! I tried running this code with Deepmind's LEO hyperparameters (https://github.com/deepmind/leo) for the miniImageNet 1-shot task, which resulted in a surpisingly low score (Meta Test Accuracy: 0.4390). Do you have any hypotheses as to why this is happening? 

Furthermore, I also ran the code for the miniImageNet 1-shot task using your hyperparameters, and got a Meta Test Accuracy of .5825, which is around 1% slightly lower than what your posted. Taking a closer look at the model during test time, I noticed that all the generated latents with these hyperparameters are mapped to 0 after the relation net. Any hypotheses here?

Thanks for your help!

**Comment by Zhen-Zohn-WANG** [comment]
setting the metatrain_batch_size as 12 could improve a little performance.
